test/apivalidations/common_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2024 The ORC Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apivalidations
+
+import (
+	"math/rand"
+
+	applyconfigv1alpha1 "github.com/k-orc/openstack-resource-controller/pkg/clients/applyconfiguration/api/v1alpha1"
+)
+
+// Create a random string of length n
+func generateResouceName(prefix string) string {
```

Comment:
You shouldn't need this, btw, as each test runs in its own namespace. (Also typo in the name, btw)

Reply:
Agree